OUTLIERDETECTION.txt

```diff
@@ -0,0 +1,43 @@
+-After loading all of the data, we set a threshold on each volume in order to find
+  the parts of the data that were outside of the brain.
+-We calculated the root mean squared difference between volumes only for the voxels
+  we found to be outside the brain.
+-All outliers below are +/- 2.5 standard deviations from the mean of these values for
+  each run.
+-The outliers below were also plotted in the graph as red dots as a means of visualizing the
+  difference between the outliers and the rest of the volumes.
+
+
+-For outlier values that are at the beginning of the run (eg. outliers numbers 0-4),
```

Comment:
I was hoping you would have a look at the outliers you detected, and see whether you could work out where the differences were, and therefore what the causes of the differences were.  For example - is the difference due to scanner drift?  Subject movement?